client/src/components/common/DownloadAlbumButton.tsx

```diff
@@ -99,44 +123,71 @@ const DownloadAlbumButton: React.FC<{
           className={css`
             display: flex;
             flex-direction: column;
+              text-align: center;
           `}
         >
-          <p>What file type do you want to download?</p>
-          <Select
-            value={chosenFormat}
-            onChange={(e) => setChosenFormat(e.target.value)}
-            options={formats.map((format) => ({
-              value: format,
-              label: format,
-            }))}
-            disabled={isGeneratingAlbum > 0}
-          />
-          <Button
-            compact
-            className={css`
-              margin-top: 0.5rem;
-              font-size: 1.2rem;
-              background: transparent;
-            `}
-            isLoading={isDownloading}
-            onClick={() => downloadAlbum()}
-            disabled={isGeneratingAlbum > 0}
-          >
-            Download
-          </Button>
-          {isGeneratingAlbum > 0 && (
-            <p
-              className={css`
-                margin-top: 1rem;
-
-                svg {
-                  margin-right: 0.5rem;
-                }
-              `}
-            >
-              <LoadingSpinner />
-              We're generating the album! Hold on for a minute...
-            </p>
+          {!chosenFormat && !isDownloading ? (
+            <>
+              <p>What file type do you want to download?</p>
```

Comment:
Wanna add this to the translations?

Reply:
I was sure I got all the strings... It's translatable now.